### PR TITLE
WIP: ClusterAwareUpdating PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1871161](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1871161)

1 new cmdlet, changes to 5 others.

### CODE REVIEW

File: docset/winserver2022-ps/clusterawareupdating/Add-CauClusterRole.md [hunk 1]
```diff
@@ -87,6 +97,24 @@ Because the command uses the *Force* parameter, the cmdlet runs without displayi
 
 ## PARAMETERS
 
+### -AttemptSoftReboot
+Indicates that the CAU clustered role attempts a Kernel Soft Reboot (KSR) for the failover cluster.
+
+KSR bypasses  BIOS/FIRMWARE initialization.
+You can only use KSR for updates that do not require a firmware/BIOS initialization.
```
Comment (PatAltimore) [inline docset/winserver2022-ps/clusterawareupdating/Add-CauClusterRole.md:104]: Can you make BIOS/FIRMWARE consistent?

File: docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md [hunk 2]
```diff
@@ -53,6 +54,24 @@ This example also shows how to pass the administrative credentials for cluster C
 
 ## PARAMETERS
 
+### -AttemptSoftReboot
+Indicates that command assumes a Kernel Soft Reboot (KSR) for the failover cluster.
+
+KSR bypasses  BIOS/FIRMWARE initialization.
+You can only use KSR for updates that do not require a firmware/BIOS initialization.
```
Comment (PatAltimore) [inline docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md:61]: Can you make BIOS/FIRMWARE consistent?

**Review by PatAltimore — CHANGES_REQUESTED** [review]

File: docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md (hunk 2 above)
Comment (TimShererWithAquent) [reply docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md:61]: Found the article that I based this on and picked a consistent wording.

**Comment by TimShererWithAquent** [comment]
I'm going to close this one to keep my queue clear. Anyone is welcome to reopen, if needed.

**Comment by PatAltimore** [comment]
Replaced with https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2870.

**Pull request closed without merging**